expression/aggregation/agg_to_pb.go

```diff
@@ -51,6 +51,8 @@ func AggFuncToPBExpr(sc *stmtctx.StatementContext, client kv.Client, aggFunc *Ag
 		tp = tipb.ExprType_Agg_BitXor
 	case ast.AggFuncBitAnd:
 		tp = tipb.ExprType_Agg_BitAnd
+	case ast.AggFuncVarPop:
```

Comment:
actually it's a dead code since we haven't pushdown `var_pop()` to the coprocessor layer.

Reply:
I have already noticed it. The code underneath the above code “client.IsRequestTypeSupported” will return nil when the case is “ast.AggFuncVarPop”. So do I need to modify the above code?